packages/core/src/exchanges/compose.ts

```diff
@@ -8,7 +8,15 @@ export const composeExchanges = (exchanges: Exchange[]): Exchange => {
 
   return payload => {
     return exchanges.reduceRight((forward, exchange) => {
-      return exchange({ client: payload.client, forward });
+      return exchange({
+        client: payload.client,
+        forward,
+        dispatchDebug: e =>
+          payload.client.debugTarget!.dispatchEvent({
```

Comment:
@kitten is this what you were thinking?

```suggestion
          process.env.NODE_ENV === "production" ? noop : payload.client.debugTarget!.dispatchEvent({
```

Reply:
Yep :+1:

Reply:
This is handled in the babel plugin now :+1: